### PR TITLE
project/serializer//userdashboard/ModerationProjects: adding addition…

### PR DESCRIPTION
**Pull request by phillimorland**
…al info for mod dashboard and adding it to component

just added dummy numbers for comment count and offensive so far

### CODE REVIEW

**Comment by phillimorland** [comment]
ok thats a big drop, will look into tests next week!

**Comment by phillimorland** [comment]
@Rineee actually got this working so feel ree to check it! :)

File: apps/projects/serializers.py [hunk 1]
```diff
@@ -10,11 +13,47 @@ class ProjectSerializer(serializers.ModelSerializer):
...
-                  'tile_image_copyright']
+                  'tile_image_copyright',
+                  'status', 'access',
+                  'participation_active',
+                  'participation_string',
+                  'future_phase', 'active_phase',
+                  'past_phase',
+                  'offensive', 'comment_count']
+
+    @lru_cache(maxsize=1)
```
Comment (Rineee) [inline apps/projects/serializers.py:36]: What does this do? Cache the status somehow?

**Comment by Rineee** [comment]
Cool, its all there and working and the tests even increased :+1: 
It looks a bit squeezed though for me, is that just for later and I merge it like this?
image

File: apps/projects/serializers.py (hunk 1 above)
Comment (phillimorland) [reply apps/projects/serializers.py:36]: This is an opteermising cache algorithum used in mB which I just copied over with some of the serializer stuff, it removes least recently used cached items, I would imagine we won't have long project lists for the mods but I would say it doesn't hurt to have it, what do you think?

**Review by phillimorland — COMMENTED** [review]
which broswers? mines looking bit different to that

**Comment by Rineee** [comment]
> which broswers? mines looking bit different to that

In firefox. I can also merge first, whatever you prefer!

File: apps/projects/serializers.py (hunk 1 above)
Comment (Rineee) [reply apps/projects/serializers.py:36]: Sure, we can keep it. Was just wondering what it does, but sounds good!

**Comment by phillimorland** [comment]
@Rineee try merging? I can't reproduce currently

**Pull request merged**